### DISCUSSION TITLE
Plastic Tool:  Provide Means to Move "Root Vertex" (Independently from Edit Tool)

### DISCUSSION THREAD

**Discussion by PsmithG** [opentoonz/opentoonz#6531] [Ideas]
Just as any vector shape can be moved independently from its "center" position (as defined by the "Edit Tool") - so should an entire "Plastic" - controlled mesh be able to be moved independently from that Center, as well.  The Plastic Skeleton Root vertex needs the ability to be moved - thus moving the entire Plastic Skeleton.

This would enable the production of characters which are made to "walk" through a scene, without first creating a "walk cycle" - and then hoping that you can move the character convincingly by pushing him through the scene by means of the "Edit Tool"

Using the latter method almost always introduces "skating" problems dealing with the feet of the character appearing to slide along the surface of the ground.

Greg Smith

**Comment by animking** [comment]
very much needed.

**Comment by psychoanima** [comment]
In my opinion root bone should act as a pin, and stay locked all the time. But with an option to switch root bones.
https://github.com/opentoonz/opentoonz_docs/issues/194

**Comment by PsmithG** [comment]
Pschoanima:

I'm afraid I don't understand your reasoning here.  Maybe you can explain why you want such a behavior to exist?

Greg Smith

**Comment by psychoanima** [comment]
Greg, 
I am sorry that you didn't catch my thought. In my feature request that I've referenced here https://github.com/opentoonz/opentoonz_docs/issues/194 I explained in details what would be perfect feature to make easy walk cycle/lifting/bone locking with plastic tool. That's the same technique that IK tool is using, and that's the reason why I am proposing consistent approach.

**Comment by ghost** [comment]
Based on the discussion in opentoonz/opentoonz#1207 feature requests are going to be closed here on GitHub unless there is a developer actively working on the feature. Feature requests can be discussed at the OpenToonz Google Group: https://groups.google.com/forum/#!forum/opentoonz_en

**Comment by RodneyBaker** [comment]
Reopening as the restriction against feature requests was long ago removed.

**Comment by RodneyBaker** [comment]
A thought on this request...

If we have an option to hide the root vertex and/or shift the root to another vertex perhaps that would meet the intent of this and similar requests.
